mesonbuild/modules/rust.py

```diff
@@ -243,12 +250,22 @@ def bindgen(self, state: ModuleState, args: T.List, kwargs: FuncBindgen) -> Modu
         else:
             name = header.get_outputs()[0]
 
+        inline_wrapper_args = []
+        outputs = [kwargs['output']]
+        if kwargs['output_inline_wrapper']:
```

Comment:
We should do a check that you have bindgen of `0.64` or later here. I think that's just giong to be `if not mesonlib.version_compare(self._bindgen_bin.get_version(), '>= 0.64'): raise MesonException(...)`